src/components/SortInput/SortInput.jsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import { ContentName, SelectStyle } from "components/SidePanel/styles";
+import { Paragraph } from "components/shared-styled-comp/shared-styled-comp";
+
+import React, { useState } from "react";
+import arrow from "assets/arrow.svg";
+import { Icon, SortContainer } from "components/SortInput/styles";
+
+/**
+ * Functional component for sorting movie results.
+ * @component
+ */
+export default function SortInput() {
+  const [isActive, setIsActive] = useState(false);
+
+  /**
+   * Toggles the display of the content.
+   * @function
+   * @param {object} e - Event object.
+   */
+  const showContent = (e) => {
+    e.preventDefault();
+    setIsActive(!isActive);
+  };
+
+  const sortOptions = [
```

Comment:
Can we move the definition of the `sortOptions` array outside of the component?  at the top of the file or in a separate constants file maybe,  with this we will avoid unnecessary recreations of the array on each render